src/playwrightTestServer.ts

```diff
@@ -226,12 +217,21 @@ export class PlaywrightTestServer {
       errorContext: { format: 'json' },
       ...runOptions,
     };
-    void connection.runTests(options);
 
+    // TODO: check if it's safe to also add this to disposables. are there cases where we want to send stopTestsNoReply after runTests is finished?
```

Comment:
There should be no such cases. If anything, it might erroneously cancel the next test run.